Makefile

```diff
@@ -39,9 +38,9 @@ lint:
 	@echo "Checking formatting..."
 	@gofmt -d -s $(PACKAGE_FILES) 2>&1 | tee lint.log
 	@echo "Checking vet..."
-	@$(foreach dir,$(PACKAGE_FILES),go tool vet $(dir) 2>&1 | tee -a lint.log;)
+	@go vet ./... 2>&1 | tee -a lint.log;)
 	@echo "Checking lint..."
-	@$(foreach dir,$(PKGS),golint $(dir) 2>&1 | tee -a lint.log;)
+	@golint . 2>&1 | tee -a lint.log;)
```

Comment:
`golint ./...`?

Reply:
> Golint checks vendor when used with "./..." so pass in "." explicitly.

:(

Reply:
`golint $(go list ./...)`? Otherwise we're only checking the top-level package (should this library ever get a subpackage).

Reply:
:+1: